UPtables/lib/asp/constraints/hard/mandatory_and_recommended_in_same_semester.lp

```diff
@@ -4,4 +4,7 @@
 
 :- not { assigned(C,D,P) : recommendation(C,S,Cu), mandatory(C,Cu) } 1, semester(S), cu(Cu), d(D), ppd(P).
 
+%%% or if they are recommended for the first three semester, to ensure that students
+%%% of the third semester can repeat the courses of the first semester
 
+:- not { assigned(C,D,P) : recommendation(C,1..3,Cu), mandatory(C,Cu) } 1, cu(Cu), d(D), ppd(P).
```

Comment:
@felixKubicek should I do another constraint here or should I leave it just like that?